blog-service/2023-04-14-deactivation-of-unused-access-keys.md

```diff
@@ -0,0 +1,18 @@
+---
+title: Deactivation of unused access keys
+image: https://www.sumologic.com/img/logo.svg
+keywords:
+  - access keys
+hide_table_of_contents: true
+authors:
+  - url: https://help.sumologic.com/release-notes-service/rss.xml
+    image_url: /img/release-notes/rss-orange.png
+---
+
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+By default API access keys will now automatically be deactivated if they have been unused for 30 days. This enhances the security of your account by reducing the risk of unauthorized access through inactive access keys. This ensures that only active access keys can be used to access your account and its resources. 
```

Comment:
Usually we preface the release note with "we'd like to announce" for releases and for behavior changes "We'd like to update you on the following change"